unblob/handlers/archive/hp/bdl.py

```diff
@@ -0,0 +1,105 @@
+import io
```

Comment:
I would add a link to the specification or where more information is available on the format at least in the commit comment.